tests/mock/mock.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
```

Comment:
Consider reusing `tests/dummy/dummy.proto` instead; feel free to add message types to that file. Alternatively, you may rename it to `mock` if that is more appropriate. You should add/update the `tests/Makefile` to compile the proto file if `mock` is the name we go with.